Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleIdentifier</key>
+	<string>com.dhinakg.SKLAsKBLGraphicsInfo</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>Intel Skylake Graphics Driver Properties Injection for macOS 13 by Dhinak G</string>
+	<key>CFBundlePackageType</key>
+	<string>KEXT</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0.0</string>
+	<key>CFBundleVersion</key>
+	<string>1.0.0</string>
+	<key>IOKitPersonalities</key>
+	<dict>
+		<key>Gen7</key>
+		<dict>
+			<key>CFBundleIdentifier</key>
+			<string>com.apple.driver.AppleIntelKBLGraphics</string>
+			<key>Development</key>
+			<dict>
```

Comment:
What values are different from KBL kext here?

Reply:
CC @dhinakg

Reply:
Well, it is easier to mention the differences from the SKL kext: 
- `Development/DisablePageFaultHandling` and `Development/DisplaySurfaceInEDRAM` were added from the KBL kext (was unsure whether they mattered, so just left them in)
- all the bundle names are changed to the KBL ones
- `IOVARendererID` is now the value from the KBL kext. 
I figured copying the values from the SKL kext would provide the most compatibility; although it's possible they dropped the old stuff, I haven't noticed any breakage yet.

The differences from the KBL kext:
- `Gen95` references changed to `Gen9`
- **`IOGVAXDecode` removed** (this was the cause of the YouTube issues - Skylake does not have VP9 hw decode)
- `IOGVAHEVCDecode` changed to `1` (KBL was `2`)
- Profile 2 removed from `IOGVAHEVCDecodeCapabilities` and `IOGVAHEVCEncodeCapabilities`
- `VTRating` in `IOGVAHEVCEncodeCapabilities` changed to `90` (KBL was `100`)

I must admit I don't know the significance of most of these changes but given they are a straight rip from the SKL kext it *should* work

Reply:
UPDATE - Resolved/Spoiled due to the abolishing of this codeless kext.